scripts/config/notebook-testing.toml

```diff
@@ -1,68 +1,83 @@
-# For notebooks to be tested "normally" (no mocking)
-notebooks_normal_test = [
-    "docs/guides/construct-circuits.ipynb",
+default-strategy="ci"
+
+[test-strategies]
+ci = { timeout = 300 }
+
+[groups]
```

Comment:
I don't think this is necessary since you're not setting anything underneath the key?

Reply:
Nice! I assumed you'd need to define the table before adding fields but it turns out you don't

```suggestion
```